.sass-lint.yml

```diff
@@ -2,12 +2,12 @@
 # Example config: https://github.com/sasstools/sass-lint/blob/master/docs/sass-lint.yml
 
 files:
-  include: 'scss/**/*.s+(a|c)ss'
+  include: "scss/**/*.s+(a|c)ss"
```

Comment:
We are using stylelint elsewhere, but probably something for another PR

Reply:
Oh, just noticed we are using stylelint in this project, so is this config still needed at all?

Reply:
Good catch, it's gone.